contrib/ipmitool/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "ipmitool"
+pkgver = "1.8.18"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-ipmievd",
+    "--enable-ipmishell",
+    "--enable-intf-lan",
+    "--enable-intf-lanplus",
+    "--enable-intf-open",
+]
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = ["automake", "gmake", "libtool"]
+makedepends = [
+    "linux-headers",
+    "openssl-devel",
+    "readline-devel",
+]
+pkgdesc = "CLI to manage IPMI systems"
+maintainer = "Renato Botelho do Couto <renato@netgate.com>"
+license = "BSD-3-Clause"
+url = "https://sourceforge.net/projects/ipmitool"
```

Comment:
```suggestion
url = "https://codeberg.org/IPMITool/ipmitool"
```

it says sourceforge isn't used anymore

this also has 1.8.19